host-bin/edit-chroot

```diff
@@ -403,7 +403,7 @@ for NAME in "$@"; do
             tardest="$dest"
             splitpid=''
         fi
-        echo -n "  Backing up $CHROOT to $dest..." 1>&2
+        echo -n "  Backing up $CHROOT to `realpath $dest`..." 1>&2
```

Comment:
I believe the crouton devs have converted most if not all syntax to replace grave accents (backticks) to the newer convention using $(...).